packages/core/src/app/customer/Customer.tsx

```diff
@@ -59,293 +46,123 @@ export interface CustomerProps {
     onWalletButtonClick?(methodName: string): void;
 }
 
-export interface WithCheckoutCustomerProps {
-    canSubscribe: boolean;
-    customerAccountFields: FormField[];
-    checkoutButtonIds: string[];
-    defaultShouldSubscribe: boolean;
-    email?: string;
-    firstName?: string;
-    forgotPasswordUrl: string;
-    isContinuingAsGuest: boolean;
-    isCreatingAccount: boolean;
-    isExecutingPaymentMethodCheckout: boolean;
-    isGuestEnabled: boolean;
-    hasBillingId: boolean;
-    isInitializing: boolean;
-    isSendingSignInEmail: boolean;
-    isSignInEmailEnabled: boolean;
-    isSigningIn: boolean;
-    privacyPolicyUrl?: string;
-    providerWithCustomCheckout?: string;
-    requiresMarketingConsent: boolean;
-    signInEmail?: SignInEmail;
-    signInEmailError?: Error;
-    isBuyNowCart: boolean;
-    isAccountCreationEnabled: boolean;
-    isPaymentDataRequired: boolean;
-    createAccountError?: Error;
-    signInError?: Error;
-    isFloatingLabelEnabled?: boolean;
-    isExpressPrivacyPolicy: boolean;
-    shouldRedirectToStorefrontForAuth: boolean;
-    clearError(error: Error): Promise<CheckoutSelectors>;
-    continueAsGuest(credentials: GuestCredentials): Promise<CheckoutSelectors>;
-    deinitializeCustomer(options: CustomerRequestOptions): Promise<CheckoutSelectors>;
-    executePaymentMethodCheckout(
-        options: ExecutePaymentMethodCheckoutOptions,
-    ): Promise<CheckoutSelectors>;
-    initializeCustomer(options: CustomerInitializeOptions): Promise<CheckoutSelectors>;
-    sendLoginEmail(params: { email: string }): Promise<CheckoutSelectors>;
-    signIn(credentials: CustomerCredentials): Promise<CheckoutSelectors>;
-    createAccount(values: CustomerAccountRequestBody): Promise<CheckoutSelectors>;
-    shouldRenderStripeForm: boolean;
-}
 
 export interface CustomerState {
     isEmailLoginFormOpen: boolean;
     isReady: boolean;
     hasRequestedLoginEmail: boolean;
+    draftEmail?: string;
 }
 
-class Customer extends Component<CustomerProps & WithCheckoutCustomerProps & AnalyticsContextProps, CustomerState> {
-    state: CustomerState = {
+const Customer: React.FC<CustomerProps & AnalyticsContextProps> = ({
```

Comment:
```suggestion
const Customer: React.FC<CustomerProps> = ({
```
Can we get AnalyticsContext from the hook?